core/server/data/schema/default-settings.json

```diff
@@ -86,7 +86,7 @@
             "defaultValue": "[{\"label\":\"Home\", \"url\":\"/\"}]"
         },
         "slack": {
-            "defaultValue": "[{\"url\":\"\"}]"
+            "defaultValue": "[{\"url\":\"\", \"username\":\"Ghost\"}]"
```

Comment:
This creates some inconsistency. Fresh blogs will have a username property and old blogs won't have a username property.

Reply:
@i8ramin ping